code/datums/intercept_text.dm

```diff
@@ -88,16 +88,15 @@
 		src.text += "discovered the following set of fingerprints ([fingerprints]) on sensitive materials, and their owner should be closely observed."
 		src.text += "However, these could also belong to a current Cent. Com employee, so do not act on this without reason."
 
-/datum/intercept_text/proc/build_rev(correct_mob)
+/datum/intercept_text/proc/build_rev(var/datum/mind/correct_mob)
```

Comment:
Is this a mind or a mob?

Reply:
its a mind, i left it as mob because every other build text proc uses mob for some reason (i was being lazy)